Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss

```
@@ -3714,8 +3714,8 @@ Item summary styling
 .itemsummary-layout #col1 .displayNodeHalf {
   display: inline-block;
   vertical-align: top;
-  margin: -15px 0;
-  width: 49%;
+  margin: -$doubleMargin $doubleMargin 0 0;
+  width: calc(50% - 32px);
```

To highlight the intention here (i.e. have it half width but take into account the margin from the two columns) would it be be possible to do:

```suggestion
  width: calc(50% - 2 * $doubleMargin);
```

---

Yup! Much cleaner. I've put that in. Thanks.